techs/zetapack/factions/magic/magic.xml

```diff
@@ -4,7 +4,7 @@
 	<starting-resources>
 		<resource name="gold" amount="500"/>
 		<resource name="wood" amount="400"/>
-		<resource name="stone" amount="400"/>
+		<resource name="stone" amount="750"/>
```

Comment:
However this may be a bit too much. We should test it.

Reply:
Ok. I'll just merge it for now so it'll be easier to test. Please let me know if you and your friends find it too much. We'll try to avoid making any more changes to magic until we get more feedback.